core/rawdb/accessors_indexes.go

```diff
@@ -64,8 +64,8 @@ func WriteTxLookupEntries(db ethdb.KeyValueWriter, block *types.Block) {
 }
 
 // DeleteTxLookupEntry removes all transaction data associated with a hash.
-func DeleteTxLookupEntry(db ethdb.KeyValueWriter, hash common.Hash) {
-	db.Delete(txLookupKey(hash))
+func DeleteTxLookupEntry(db ethdb.KeyValueWriter, hash common.Hash) error {
+	return db.Delete(txLookupKey(hash))
```

Comment:
👍

Reply:
No, it's not good. Do `log.Crit` if you must. The same as with writes. Why? Because we shot ourselves it the foot all over by always ignoring database write errors, or littering the entire code with just log.Crit at the call sites.